kube-config-files/methode-content-placeholder-mapper.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: methode-content-placeholder-mapper
+  labels:
+    app: methode-content-placeholder-mapper
+    visualize: "true"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: methode-content-placeholder-mapper
+  template:
+    metadata:
+      labels:
+        app: methode-content-placeholder-mapper
+        visualize: "true"
+    spec:
+      containers:
+      - name: methode-content-placeholder-mapper
+        image: coco/methode-content-placeholder-mapper:remove-host-routing
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 256Mi
+        livenessProbe:
+          initialDelaySeconds: 15
```

Comment:
Is there a reason this value can't be lowered to 5 seconds? If this one can be lowered the value from readinessProbe could be lower as well.